weave/trace_server/calls_query_builder.py

```diff
@@ -209,7 +258,8 @@ def as_sql(self, pb: ParamBuilder, table_alias: str) -> str:
         if self._consumed_fields is None:
             self._consumed_fields = []
             for field in conditions.fields_used:
-                self._consumed_fields.append(get_field_by_name(field))
+                # TODO: Verify that this is "ok" since before we were just looking at field name
```

Comment:
didn't we change `get_field_by_name` in this pr to make this work?

Reply:
👍